src/cmdc_tools/datasets/official/FL/data.py

```diff
@@ -0,0 +1,45 @@
+import pandas as pd
+import us
+
+from ...base import DatasetBaseNoDate
+from ..base import ArcGIS
+
+
+class Florida(DatasetBaseNoDate, ArcGIS):
+    ARCGIS_ID = "CY1LXxl9zlJeBuRZ"
+    state_fips = int(us.states.lookup("Florida").fips)
+    has_fips: bool = True
+    source = "https://fdoh.maps.arcgis.com/apps/opsdashboard/index.html#/8d0de33f260d444c852a615dc7837c86"
+
+    def get(self):
+        df = self.get_all_sheet_to_df("Florida_COVID19_Cases_by_County_vw", 0, 1)
+        renamed = df.rename(
+            columns={
+                "COUNTY": "fips_prefix",
+                "TPositive": "positive_tests_total",
```

Comment:
@tlyon3 -- Did you double check these numbers?

`TPositive` and `TNegative` don't seem to line up with the numbers from the dashboard, but `T_Positive` and `T_Negative` do. I'm going to make this change, but will you look into this in the morning?